.yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
```

Comment:
@dmuharemagic this plugin is required for publishing all at once ?

Reply:
Yes, that is what enables iterative operations on the workspaces.